worker.py

```
@@ -0,0 +1,192 @@
+def main():
+    """Entry point for worker"""
+    # Register signal handlers
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    
+    # Run worker
+    try:
+        asyncio.run(worker())
+    except KeyboardInterrupt:
+        logger.info("Worker interrupted by user")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)